CHANGELOG.md

```diff
@@ -4,6 +4,188 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.46.0] - 2024-05-07
+
+## 🚀 Features
+
+### Entity cache preview: support queries with private scope ([PR #4855](https://github.com/apollographql/router/pull/4855))
+
+**This feature is part of the work on [subgraph entity caching](https://www.apollographql.com/docs/router/configuration/entity-caching/), currently in preview.**
+
+The router now supports caching responses marked with `private` scope. This caching currently works only on subgraph responses without any schema-level information.
+
+For details about the caching behavior, see [PR #4855](https://github.com/apollographql/router/pull/4855) 
+
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/4855
+
+### Add support of custom events defined by YAML for telemetry ([Issue #4320](https://github.com/apollographql/router/issues/4320))
+
+Users can now [configure telemetry events via YAML](https://www.apollographql.com/docs/router/configuration/telemetry/instrumentation/events/). 
```

Comment:
@BrynCooke I just saw this, but can you **remove** this trailing period as soon as you merge this into `1.46.0`? I don't want to offer it on this PR because it'll require re-running CI.